server/util_test.go

```diff
@@ -431,7 +420,7 @@ func TestStartCmdwithEmptyandNonEmptyMinGasPrices(t *testing.T){
 		t.Fatalf("function failed with [%T] %v", err, err)
```

Comment:
The panic is resolved. However, there is new error arised. 
``` 
/home/ai/go/src/github.com/cosmos/cosmos-sdk/server/util_test.go:420: function failed with [*fmt.wrapError] couldn't read GenesisDoc file: open /tmp/TestStartCmdwithEmptyandNonEmptyMinGasPrices799694772/001/config/genesis.json: no such file or directory
```  
does anyone have an idea how to fix it?

Reply:
I pushed a commit where I added the `InitCmd` in the test to create this file.